core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -305,39 +305,36 @@ service MgmtPublicService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Utils"};
   }
 
-  // List pipeline triggers
+  // Get pipeline trigger count
   //
-  // Returns a paginated list of pipeline executions.
-  rpc ListPipelineTriggerRecords(ListPipelineTriggerRecordsRequest) returns (ListPipelineTriggerRecordsResponse) {
-    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/triggers"};
+  // Returns the pipeline trigger count of a given requester within a timespan.
+  // Results are grouped by trigger status.
+  rpc GetPipelineTriggerCount(GetPipelineTriggerCountRequest) returns (GetPipelineTriggerCountResponse) {
+    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/trigger-count"};
+    option (google.api.method_signature) = "requester,start,stop";
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
   }
 
-  // List pipeline trigger metrics
+  // List pipeline trigger time charts
   //
-  // Returns a paginated list of pipeline executions aggregated by pipeline ID.
-  rpc ListPipelineTriggerTableRecords(ListPipelineTriggerTableRecordsRequest) returns (ListPipelineTriggerTableRecordsResponse) {
-    option (google.api.http) = {get: "/v1beta/metrics/vdp/pipeline/tables"};
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
-  }
-
-  // List pipeline trigger computation time charts
-  //
-  // Returns a paginated list with pipeline trigger execution times, aggregated
-  // by pipeline and time frames.
+  // Returns a timeline of pipline trigger counts for a given requester. The
+  // response will contain one set of records (datapoints), representing the
```

Comment:
ℹ️ Right now the chart will contain one line with _all the triggers_ executed by a given requester. In the future we might group counts by other fields like pipeline ID, status, trigger mode, etc., in which case one datapoint collection per category will be returned.